h2o-py/tests/testdir_parser/pyunit_parquet_adjust_timezone.py

```diff
@@ -0,0 +1,39 @@
+import sys
+
+sys.path.insert(1, "../../../")
+from tests import pyunit_utils
+from h2o.frame import H2OFrame
+import tempfile
+import h2o
+
+'''
+Adjust timestamp parquet
+'''
+
+test_local_user_timezone = "America/Chicago"
+
+
+def adjust_timestamp_parquet():
+    with tempfile.TemporaryDirectory() as dir:
+        # prepare the file which will be imported
+        input_timestamp = '2024-08-02 12:00:00'
+        original_timestamp_df = H2OFrame({"timestamp": input_timestamp})
+        h2o.export_file(original_timestamp_df, path=dir + "/import", format="parquet", write_checksum=False)
+
+        # import the file and see tz_adjust_to_local works
+        imported_df = h2o.import_file(dir + "/import", tz_adjust_to_local=True)
+        expected_df = H2OFrame({"timestamp": '2024-08-02 07:00:00'})
```

Comment:
This test is failing for me.  I am running it on PST.  So you need to adjust this answer to the local cluster time zone and expected_df should contain '2024-08-02 12:00:00' adjusted dynamically to the local cluster timezone.